parsec/core/cli/bootstrap_organization.py

```diff
@@ -26,21 +29,30 @@ async def _bootstrap_organization(config, addr, password, force):
             )
 
         device_display = click.style(new_device.slughash, fg="yellow")
+        # We don't have to worry about overwritting an existing keyfile
+        # given their names are base on the device's slughash which is intended
+        # to be globally unique.
         with operation(f"Saving device {device_display}"):
             save_device_with_password(
-                config_dir=config.config_dir, device=new_device, password=password, force=force
+                config_dir=config.config_dir, device=new_device, password=password
```

Comment:
> We don't have to worry about overwritting an existing keyfile
> given their names are base on the device's slughash which is intended
> to be globally unique.

Maybe the `force` argument should also be removed from `save_device_with_password` then?

Reply:
👍 8a666202d3c55cf13ca241a450cc22f182ecbad4